frontend/src/components/camp/Picasso.vue

```diff
@@ -330,7 +330,7 @@ export default {
           if (!this.extendOriginal) {
             this.showEntryInfoPopup(this.currentEntry)
           }
-        } else {
+        } else if (this.currentEntry.length !== this.currentEntry._storeData.length) {
```

Comment:
Since there was no drag (then the if above would be true => no else): if there was no change in length there is no need to update.